### ISSUE TITLE
https://developer.anchorfree.com/ is not loading error 503 

### ISSUE THREAD

**Issue by Akhandsaxena12** [AnchorFreePartner/hydrasdk-demo-android#176]
503 Service Unavailable
No server is available to handle this request.

**Comment by goldfish07** [comment]
use [https://developer.aura.com](https://developer.aura.com)/ instead

**Comment by Akhandsaxena12** [comment]
> use https://developer.aura.com/ instead

How to signup sir

**Comment by goldfish07** [comment]
> 



> > use https://developer.aura.com/ instead
> 
> How to signup sir

no need to signup, just use the same login credentials that you used for developer.anchorfree.com

**Comment by Akhandsaxena12** [comment]
> > 
> 
> > > use https://developer.aura.com/ instead
> > 
> > 
> > How to signup sir
> 
> no need to signup, just use the same login credentials that you used for developer.anchorfree.com

Sir but I need to make new account I do not have any account :(

**Comment by goldfish07** [comment]
oh! you don't have an account yet, I see there is no signup portal in [https://developer.aura.com/](https://developer.aura.com/).
for now, you can do nothing but wait!

**Comment by Akhandsaxena12** [comment]
> > 
> 
> > > use https://developer.aura.com/ instead
> > 
> > 
> > How to signup sir
> 
> no need to signup, just use the same login credentials that you used for developer.anchorfree.com
 I need urgently

**Comment by DeePsycho** [comment]
cant login with same cardentials,
password error, 
when tried to reset.
Reset link has been expired
every time same issue

**Comment by pixymobileco** [comment]
No, same crediantal not work. is there a bug? Also pass reset expired everytime 


@goldfish07

**Comment by goldfish07** [comment]
![Screenshot from 2022-05-06 20-43-23](https://user-images.githubusercontent.com/34901152/167162329-850f59a8-83a9-47cb-b60d-29b51fe7290a.png)
@pixymobileco 
@DeePsycho

**Comment by pixymobileco** [comment]
Oh thats good but, i cant change pass cause give me expired link

**Comment by DeePsycho** [comment]
Email and password do not match
same error

**Comment by pixymobileco** [comment]
@goldfish07 hello Email and password do not match my anchorfree user doesn't login same crediantals. I thinks its bug.

**Comment by DeePsycho** [comment]
Yup
You're correct

**Comment by bhavin-qfonapp** [comment]
how to do new signup?

**Comment by numan1232** [comment]
> use https://developer.aura.com/ instead

Why can't I create a new account on this website? Help

**Comment by numan1232** [comment]
> > > 
> > 
> > 
> > > > use https://developer.aura.com/ instead
> > > 
> > > 
> > > How to signup sir
> > 
> > 
> > no need to signup, just use the same login credentials that you used for developer.anchorfree.com
> 
> Sir but I need to make new account I do not have any account :(

**Comment by numan1232** [comment]
> use https://developer.aura.com/ instead

Why can't I create a new account on this website?

**Comment by DeePsycho** [comment]
You still want?

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by nilpatel9099sn** [comment]
Anyone know how to create new account on  https://developer.aura.com/ ??
